pkg/registry/core/event/strategy.go

```diff
@@ -84,19 +84,20 @@ func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
 	if !ok {
 		return nil, nil, fmt.Errorf("not an event")
 	}
-	return labels.Set(event.Labels), EventToSelectableFields(event), nil
+	return labels.Set(event.Labels), ToSelectableFields(event), nil
 }
 
-func MatchEvent(label labels.Selector, field fields.Selector) storage.SelectionPredicate {
+// Matcher returns a generic matcher for a given label and field selector.
```

Comment:
s/a generic matcher/a selection predicate/

Reply:
Updated, also for [pkg/registry/core/configmap/strategy.go](https://github.com/kubernetes/kubernetes/blob/117a0e525d72be56cd18c2c16dd5c0c8fd8baf0b/pkg/registry/core/configmap/strategy.go#L98) and [pkg/registry/core/configmap/strategy.go](https://github.com/kubernetes/kubernetes/blob/117a0e525d72be56cd18c2c16dd5c0c8fd8baf0b/pkg/registry/core/configmap/strategy.go#L98) which had the same description. (I hope that doesn't mean we have to pull in more reviewers..)